charts/falco/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 This file documents all notable changes to Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v4.14.2
+* fix(falco/readme): use `rules_files` instead of deprecated `rules_file` in README config snippet
+
 ## v4.14.1
 * fix(falco/dashboard): make pod variable independent of triggered rules. CPU and memory are now visible for each 
   pod, even when no rules have been triggered for that falco instance.
```

Comment:
```suggestion
## v4.14.2

* fix(falco/readme): use `rules_files` instead of deprecated `rules_file` in README config snippet

## v4.14.1

* fix(falco/dashboard): make pod variable independent of triggered rules. CPU and memory are now visible for each 
  pod, even when no rules have been triggered for that falco instance.
```

Reply:
Pushed again with suggested formatting fix